lib/web_ui/lib/src/engine/view_embedder/flutter_view_manager.dart

```diff
@@ -101,5 +110,49 @@ class FlutterViewManager {
     // close the streams.
     _onViewCreatedController.close();
     _onViewDisposedController.close();
+    // Remove the focus tracking event listeners.
+    domDocument.body?.removeEventListener(_focusin, _focusChangeHandler, true);
+    domDocument.body?.removeEventListener(_focusout, _focusChangeHandler, true);
+  }
+
+  int? _latestFocusedViewId;
+
+  // TODO(tugorez): Add handlers for backwards focus.
+  late final DomEventListener _focusChangeHandler = createDomEventListener((DomEvent event) {
+    final int? viewId = _findParentViewIdOrNull(domDocument.activeElement);
+    if (viewId == _latestFocusedViewId) {
+      return;
+    }
+
+    late final ui.ViewFocusEvent event;
```

Comment:
This is obscuring the incoming `DomEvent event` from the callback, from which we may be able to retrieve useful information!

For example, `focusin event.relatedTarget` has "The Element losing focus, if any" and `focusout event.relatedTarget` "the Element gaining focus, if any".

Reply:
I'm not sure I follow what's the suggestion here. 

We already know which one was the previously focused view id (none as option).  What would be the advantage of using these bits?